great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -480,6 +491,31 @@ def get_domain_records(
 
         return data
 
+    def _combine_row_conditions(
+        self, row_conditions: List[RowCondition]
+    ) -> RowCondition:
+        """Combine row conditions using AND if type_ is SPARK_SQL
+
+        Args:
+            row_conditions: Row conditions of type Spark
+
+        Returns:
+            Single Row Condition combined
+        """
+        assert all(
+            [
+                condition.type_ == RowConditionParserType.SPARK_SQL
+                for condition in row_conditions
+            ]
+        ), "All row conditions must have type SPARK_SQL"
+        raw_conditions: List[str] = [
```

Comment:
@anthonyburdi Minor -- any chance to come up with a slightly different sounding/spelling variable name?  This one seems too close and thus might be error prone for potential typos.

Reply:
+1

I assumed it was the same variable but only caught it after Alex's comment. Maybe just `conditions`?

Reply:
Good call - will change 👍

Reply:
Done!